tasks/main.yml

```diff
@@ -8,9 +8,17 @@
   unarchive: src=/usr/local/src/pgweb_{{ pgweb_version }}_{{ pgweb_os }}_{{ pgweb_arch }}.zip
              dest=/usr/local/bin
              copy=no
-             creates=/usr/local/bin/pgweb
+             creates=/usr/local/bin/pgweb_{{ pgweb_version }}_{{ pgweb_os }}_{{ pgweb_arch }}
```

Comment:
Doesn't this actually create `/usr/local/bin/pgweb_{{ pgweb_os }}_{{ pgweb_arch }}`?